src/Symfony/Bundle/FrameworkBundle/composer.json

```
@@ -51,6 +51,36 @@
         "symfony/messenger": "^5.2",
         "symfony/mime": "^4.4|^5.0",
         "symfony/notifier": "^5.3",
+        "symfony/allmysms-notifier": "^5.3",
+        "symfony/clickatell-notifier": "^5.3",
+        "symfony/discord-notifier": "^5.3",
+        "symfony/esendex-notifier": "^5.3",
+        "symfony/fake-chat-notifier": "^5.3",
+        "symfony/fake-sms-notifier": "^5.3",
+        "symfony/firebase-notifier": "^5.3",
+        "symfony/free-mobile-notifier": "^5.3",
+        "symfony/gatewayapi-notifier": "^5.3",
+        "symfony/gitter-notifier": "^5.3",
+        "symfony/google-chat-notifier": "^5.3",
+        "symfony/infobip-notifier": "^5.3",
+        "symfony/iqsms-notifier": "^5.3",
+        "symfony/light-sms-notifier": "^5.3",
+        "symfony/linked-in-notifier": "^5.3",
+        "symfony/mattermost-notifier": "^5.3",
+        "symfony/microsoft-teams-notifier": "^5.3",
```

As soon as the repository symfony/message-bird-notifier is available: 
```suggestion
        "symfony/message-bird-notifier": "^5.3",
        "symfony/microsoft-teams-notifier": "^5.3",
```